satellite/console/consoleweb/server.go

```diff
@@ -156,8 +200,8 @@ func (server *Server) bucketUsageReportHandler(w http.ResponseWriter, r *http.Re
 	if err != nil {
 		server.log.Error("bucket usage report error", zap.Error(err))
 
-		w.WriteHeader(http.StatusUnauthorized)
-		http.ServeFile(w, r, filepath.Join(server.config.StaticDir, "static", "errors", "404.html"))
+		//TODO: when new error pages will be created - change http.StatusNotFound on http.StatusUnauthorized
+		server.serveError(w, r, http.StatusNotFound)
```

Comment:
note there's above `http.ServeFile` which also should probably use `server.serveError`

Reply:
PS: there shouldn't be a `http.ServeFile` anywhere here... this ensures we don't depend on FS directly at the server level.